### ISSUE TITLE
sp_Blitz - Trace File Issue

### ISSUE THREAD

**Issue by RichBenner** [BrentOzarULTD/SQL-Server-First-Responder-Kit#1761]
**Do you want to request a *feature* or report a *bug*?**
Bug. 

**What is the current behavior?**

I'm on a client server and somebody's been messing with the trace files by the looks of it. When it gets to the appropriate line it sets @TraceFileIssue to 1. This is expected, but it's then throwing errors on check number 106 (which uses this trace file). 

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via http://sqlfiddle.com**

It's easy enough to add in a line somewhere that changes @base_tracefilename to an invalid location, somewhere before the check for @TraceFileIssue. 

**What is the expected behavior?**

Well, it'd be nice if it didn't throw this error :). I've added AND TraceFileIssue = 0 to check 106 to fix it in my instance.

**Which versions of SQL Server and which OS are affected by this issue? Did this work in previous versions of our procedures?**

This is on SQL 2012 Standard (11.0.7001) but I'd imagine it's not an issue with the version specifically.

**Comment by BlitzErik** [comment]
@RichBenner were you going to work on this one? Doing some issue cleanup before the next release.

**Comment by RichBenner** [comment]
Yep, was just waiting for the ok. I’ll submit code first thing tomorrow for
this :)

**Comment by BlitzErik** [comment]
No need to wait for us!
